int64 in
string path
---
bool success
